src/components/MapView/DateSelector/index.test.tsx

```diff
@@ -6,10 +6,16 @@ import DateSelector from '.';
 import { store } from '../../../context/store';
 
 test('renders as expected', () => {
+  const realDateNow = Date.now.bind(global.Date);
```

Comment:
I was thinking about mocking the whole store

This shouldn't be a problem anymore because the initial values of dates are number

Reply:
This is a problem because the maxDate keeps changing